irohad/ametsuchi/impl/flat_file/flat_file.cpp

```diff
@@ -21,96 +21,80 @@
 #include <boost/range/adaptor/indexed.hpp>
 #include <boost/range/algorithm/find_if.hpp>
 #include <iomanip>
+#include <iostream>
 #include <sstream>
 #include "common/files.hpp"
 
-using namespace iroha::ametsuchi;
+namespace iroha {
+  namespace ametsuchi {
+    const uint32_t DIGIT_CAPACITY = 16;
```

Comment:
There is an idea for improving coverage. You should move `DIGIT_CAPACITY` into the class as a static constant field. Pls, check the coverage before/after and make a report about it.